### PR TITLE
Change styles of event.

### PR DESCRIPTION
**Pull request by farellano0**
## Quick Info

- **Feature**: Change style of event information is displayed.
- **Impact**: Frontend and Backend
- **Affected Components**: index and user's events view.

## What does this change?

- Change the table style to card style to see the event's information. Furthermore, add a responsive display.
- While editing styles, we found these errors such as:
  - Wrong redirections in the routes.
  - Validation of privacy statement and set a default value.
  - Check if the image is representable and resizable.

## Why are you changing that?

Implementing these features because the table doesn't match with the application's style.

## Merge/Deploy Checklist
- [x] Events must be visible on the cards.

## How do you manually test this?

To manually test these changes, follow these steps:
1. Navigate to the application and visualize the events created.

## Screenshots
**Desktop**
Captura desde 2024-02-13 19-20-27

**Mobile**
image

## Errors
We tried to add a validation for active storage where only accept images on jpeg/png files and when upload another kind of file, we got this error:

image

### CODE REVIEW

File: app/assets/stylesheets/base/_reset.scss [hunk 1]
```diff
@@ -9,3 +8,8 @@ body {
   flex-direction: column;
   justify-content: space-between;
 }
+
+
+h1, h2, h3, h4, h5, h6 {
+  margin: 0;
+}
```
Comment (MauricioNC) [inline app/assets/stylesheets/base/_reset.scss]: Add an empty line at the end of your code

File: app/assets/stylesheets/components/_cards.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+@include desktop-lg {
+  .cards-container {
+    flex-wrap: wrap;
+    flex-direction: row;
+
+    .card {
+      width: 30%;
+    }
+  }
+}
```
Comment (MauricioNC) [inline app/assets/stylesheets/components/_cards.scss]: Add an empty line at the end of your code

File: app/assets/stylesheets/layout/_header.scss [hunk 3]
```diff
@@ -19,11 +20,12 @@ header {
...
   }
 }
 
 @include desktop-lg {
-  header i {
-    display: none;
+  header {
+    flex-direction: row;
   }
 }
```
Comment (MauricioNC) [inline app/assets/stylesheets/layout/_header.scss]: Add an empty line at the end of your code

**Review by MauricioNC — CHANGES_REQUESTED** [review]

File: app/views/bright_events/index.html.erb [hunk 4]
```diff
@@ -5,42 +5,25 @@
...
+            <h5><%= event.location %></h5>
+          </div>
+          <p><%= event.description %></p>
+        </div>
+        <div class="card-footer">
+          <p>$ <%= event.cost %> MX</p>
+          <p><%= event.privacy ? 'Privado' : 'Público' %></p>
+        </div>
+      </div>
+    <% end %>
```
Comment (MauricioNC) [inline app/views/bright_events/index.html.erb]: Just as observation, you have almost the same code here and in the user_events.html.erb, may be you can break this in a partial

Comment (juperameza) [reply app/views/bright_events/index.html.erb]: Completely agree!!
I thinks its a must do not a suggection

**Review by juperameza — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/base/_reset.scss (hunk 1 above)
Comment (Zama15) [reply app/assets/stylesheets/base/_reset.scss]: done

File: app/assets/stylesheets/components/_cards.scss (hunk 2 above)
Comment (Zama15) [reply app/assets/stylesheets/components/_cards.scss]: done

File: app/assets/stylesheets/layout/_header.scss (hunk 3 above)
Comment (Zama15) [reply app/assets/stylesheets/layout/_header.scss]: done

File: app/views/bright_events/index.html.erb (hunk 4 above)
Comment (Zama15) [reply app/views/bright_events/index.html.erb]: okay it is not easy but i will look how to deal with it :+1:

**Review by MauricioNC — APPROVED** [review]

**Review by juperameza — APPROVED** [review]

**Pull request merged**